### PR TITLE
Updated Indentation Documentation

### PR DESCRIPTION
**Pull request by GinoMan**
- Clarified that the "same" indentation refers to same kind
- Added a paragraph that recipe bodies can be further indented
- Added an example justfile but using powershell
- Added example output for the justfile being invoked in powershell

### CODE REVIEW

**Comment by GinoMan** [comment]
Fun fact. This is my first pull request to an open source software project ever.
🎉 🥳

**Comment by casey** [comment]
Nice, thanks for the PR, and sorry for letting it sit for so long. Merging! I reworded it slightly and made the example more compact.

**Comment by GinoMan** [comment]
YAY! Thank you so much! It's great. I can tend to be a bit wordy so I'm glad you were able to shorten it and yet keep the same information. 

Thanks again!
